README.md

```diff
@@ -21,7 +21,7 @@ Thus, the package's main foci are to:
 | fivetran\_log\_connenector\_status        | Each record represents a connector loading data into a destination, enriched with data about the connector's status and the status of its data flow.                                          |
 | fivetran\_log\_transformation\_status     | Each record represents a transformation, enriched with data about the transformation's last sync and any tables whose new data triggers the transformation to run. |
 | fivetran\_log\_mar\_table\_history     | Each record represents a table's active volume for a month, complete with data about its connector and destination.                             |
-| fivetran\_log\_credit\_mar\_history    | Each record represents a destination's consumption, via its MAR, total credits used, and credits per millions MAR.                             |
+| fivetran\_log\_credit\_mar\_history    | Each record represents a destination's consumption by showing its MAR, total credits used, and credits per millions MAR.                             |
```

Comment:
Just a note: We try to avoid using "via" in our docs because it has several different definitions ("by way of/through" or "by means of")

Reply:
Ah that makes sense -- good to know!